test/users.test.js

```diff
@@ -335,13 +344,23 @@ describe('Users', () => {
           done();
         });
     });
+  });
+
+  describe('/POST users/agency', () => {
+    it('it should redirect to login if not logged in', (done) => {
+      agent.get('/users/agency').end((err, res) => {
+        res.should.have.status(200);
+        res.text.should.contain('Sign Up to Donate Gifts');
+        done();
+      });
+    });
 
-    it('it should not login without verified email', (done) => {
+    it('it should show agency registration page after initial registration for partners', (done) => {
+      signupRequest.userRole = 'partner';
       agent
         .post('/users/signup')
         .send(signupRequest)
         .end((err, res) => {
-          res.should.have.status(200);
```

Comment:
why should it not have 200?

Reply:
Oops; accidentally removed that. Adding it back :+1: